packages/@n8n/task-runner/src/js-task-runner/js-task-runner.ts

```diff
@@ -105,6 +105,43 @@ export class JsTaskRunner extends TaskRunner {
 		});
 	}
 
+	private preventPrototypePollution() {
+		if (process.env.NODE_ENV === 'test') return; // needed for Jest
+
+		[
```

Comment:
We could get all the built-ins using `Object.getOwnPropertyNames(globalThis)` instead of hardcoding them

Reply:
Almost, see the comment